examples/service-clients/azure-client/external-controller/webpack.test.cjs

```diff
@@ -14,6 +14,9 @@ module.exports = (env) => {
 		},
 		resolve: {
 			extensions: [".ts", ".tsx", ".js"],
+			extensionAlias: {
+				".js": [".ts", ".tsx", ".js", ".cjs", ".mjs"],
+			},
```

Comment:
@sonalideshpandemsft - This is something that stumped me for a while.  Adding 'extensionAlias' to 'webpack.config.cjs' seemed to fix things (i.e., it worked in npm run start), but jest/puppeteer still reported not being able to resolve modules.

Turns out the 'webpack.test.cjs' doesn't inherit from the root 'webpack.config.js'.

Reply:
does this also fix the attw failures?

Reply:
It was the incorrect export paths (which have now been removed, per Jason's comment.)